src/App.css

```diff
@@ -13,6 +13,13 @@
 
 * {
   box-sizing: border-box;
+  /* Measurements */
+}
+
+@media screen and (max-width: 730px) {
+  * {
+    --sidebar-width: calc(var(--sidebar-width) - 1em);
```

Comment:
I'm not sure what this is doing, maybe it should be applied only to the body? I think it's more expensive for the browser to apply this css variable to every element

apart from that I'm not sure what is this used for

Reply:
This is used when the sidebar is not collapsed and the screen is too small, it makes the uncollapsed sidebar a bit smaller in width.
I think it's kind of useless now because if the view is too small makes more sense to just collapse the sidebar.